lib/models/blueprint.js

```diff
@@ -1329,8 +1329,21 @@ Blueprint.list = function(options) {
 };
 
 /**
+  Files that are renamed when installed into the target directory.
+  This allows including files in the blueprint that would have an effect
+  on another process, such as a file named `.gitignore`.
+
   @static
-  @property renameFiles
+  @property renamedFiles
+
+  @example
+  ```js
+  // Rename the `gitignore` file to `.gitignore` upon installation
+  // in order to prevent it from interfering with [Git](https://git-scm.com/).
+  Blueprint.renamedFiles = {
+    'gitignore': '.gitignore'
+  };
```

Comment:
@les2 this is a `static` property, not something that you can actually set on a custom blueprint, so I'm not sure how valuable that example actually is.